src/controllers/trips.controller.js

```diff
@@ -35,15 +36,16 @@ class Trips {
       });
     });
 
+    const requestId = await createRequest(currentUser.userId, 'single');
     const trip = await db.trip.create({
       leavingFrom,
       goingTo,
       travelDate,
       returnDate,
       reason,
       hotelId,
+      requestId,
       userId: currentUser.userId,
-      status: 'pending',
       type: 'return'
     });
```

Comment:
Why not create a trip service and wrap this in a method?

Reply:
I think that is a better way to organizer the controller. Let me work on that thank you.